roles/minio/defaults/main.yaml

```diff
@@ -8,6 +8,4 @@ dojot_minio_port: 9000
 
 dojot_minio_cluster_size: 1
 
-dojot_minio_persistent_volumes: false
-
-dojot_minio_volume_size: 4Gi
+dojot_minio_volume_size: 2Gi
```

Comment:
Shouldn't we have a dojot_minio_version variable too?

Reply:
Ok, that has been changed.